src/js/modules/plant/overlay.js

```diff
@@ -69,17 +69,14 @@ export default View.extend({
       const $img = $container.children('img');
       const height = $img.height();
       const width = $img.width();
-      const pos = $container.position();
-      let top = pos.top;
-      const left = pos.left * scale;
+      const pos = object.moveable.getPosition();
```

Comment:
Yep, I see where You're using this `staticMode` pointed above - I think that better aproach will be:

- separating actually planted objects from objects available in tollbox (or palette, that's the name introduced in streetmix)

- we should add moaveble API to all planted objects,

- we should create separate draggable API to move objects durring dragging.